automatic/python3/python3.json

```diff
@@ -1,4 +0,0 @@
-﻿{
```

Comment:
should probably not remove the json file, otherwise it won't be commited when the package is updated.
Correct me if I'm wrong though.

Reply:
I thought it would... not sure. I guess that we can find out this way :)

Reply:
Unless you have changed it since I last had a look at the `git` AU plugin, you use the argument `-u` which have the following description
> -u, --update          update tracked files

This also **only** updates tracked files, and nothing else.